packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1638,5 +1638,11 @@ idx.error.code.access_denied.device_assurance.remediation.chrome.chromeos.upgrad
 idx.error.code.access_denied.device_assurance.remediation.chrome.chromeos.key_trust_level.verified_mode = Switch your device to verified mode
 idx.error.code.access_denied.device_assurance.remediation.chrome.chromeos.key_trust_level.developer_mode = Switch your device to developer mode
 
+# Device Assurance Grace Period
+idx.device_assurance.grace_period.title = Device assurance reminder
+idx.device_assurance.grace_period.warning.title.due_by_days = Your device doesn't meet the security requirements. Fix the issue within {0} days to prevent lockout.
+idx.device_assurance.grace_period.warning.title.due_by_date = Your device doesn't meet the security requirements. Fix the issue by {0} to prevent lockout.
+idx.device_assurance.grace_period.continue_to_app = Continue to app
+
```

Comment:
I think we need to update `7.21` somehow with the translations merged into `master`

Reply:
As in get the translations that I already merged into `master` into `7.21` before this PR gets merged into `7.21`? Since these translation changes are identical to the ones in `master`, won't it just resolve to a no-op once this PR is merged in and we compare `7.21` to `master`?

Reply:
There will probably be a merge conflict for two reasons:
* The translations in master had the additional comments, so the line changes aren't exactly the same
* I think git will see the lines added two different times so it won't know which set of lines to use (even though _we_ know they are the same). Not a big deal, but something to watch out for when we handle the merge of `7.21` to `master` cc: @lesterchoi-okta 

I think the only way to have avoided this would have been to add the translations to `7.21` early enough so that it would get into a `7.21.*` release, the commits would end up in `master` after the backport process, and get picked up in time for translations.

Reply:
Oh right, I can at least make the merge easier by adding the same hint comments here